apps/e2e/forms-storybook/src/app/file-attachment/file-attachment.component.html

```diff
@@ -30,6 +32,7 @@
 
 <div style="width: 60%" *ngIf="basic">
   <sky-file-drop
+    helpPopoverTitle="This should not show up"
     labelText="File attachment links removed"
     [acceptedTypes]="acceptedTypes"
     [allowLinks]="false"
```

Comment:
missing scenarios: 
- easy mode file attachment without help inline
- easy mode file attachment with required asterisk and help inline

Reply:
On the `basic` screenshot there's a third file attachment without help inline. Added required to the `no-preview` screenshot.